### ISSUE TITLE
Updating Readme.md documentation of priority repos

### ISSUE THREAD

**Issue by nathanfletcher** [unicef/publicgoods-roadmap#55]
As more code is added to a number of DPG reops and as more issues are discovered and fixed, it would be necessary at a number of points in time to update the technical documentation of key repositories.

Current top  repos with focused attention on this are:

- [submission-digitalpublicgoods](https://github.com/lacabra/submission-digitalpublicgoods)
- [publicgoods-scripts](https://github.com/unicef/publicgoods-scripts)

@lacabra Justin showed me [this PR](https://github.com/unicef/publicgoods-candidates/pull/64) about Hugo docs. I am looking into it to see how it would help this issue.

**Comment by lacabra** [comment]
Agreed on the proposed scope of work, but the PR does not directly address the issue at hand. The PR "beautifies" the existing documentation and makes it more accessible, but the issue you are uncovering is missing technical documentation.
